app/spike/internal/cmd/policy/README.md

```diff
@@ -0,0 +1,160 @@
+# SPIKE Policy Commands
```

Comment:
Not right now, but later we might want to move this to the documentation website (spike.it)

We might also want to add similar documentation to other `spike` CLI commands too.